lib/internal/http2/core.js

```diff
@@ -510,6 +510,7 @@ function onSettings() {
     return;
   session[kUpdateTimer]();
   debugSessionObj(session, 'new settings received');
+  session[kRemoteSettings] = undefined;
```

Comment:
If I'm not wrong, this line was deleted by accident in https://github.com/nodejs/node/pull/29122/files#diff-696b2cc418addca5f3fe5020058f8b15L416

/cc @addaleax